content/general/concepts/what-is-cloud-computing/what-is-cloud-computing.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Moving forward, can we name the branch after the change(s) we're making? (e.g. `added-cloud-computing-general-entry`, etc.)

Reply:
sure